releases/release-1.24/release-notes/release-notes-draft.json

```diff
@@ -1,4 +1,23 @@
 {
+  "100424": {
+    "commit": "ba1c42892f5ac079f099fa96f30f9f1b40d1283a",
+    "text": "\"NONE\"",
```

Comment:
No text for this one - is that okay?

Reply:
Hey, I've checked the related PR https://github.com/kubernetes/kubernetes/pull/100424 and it seems good to keep it? `Add test cases of kubelet_pods_test.go` What do you think